### PR TITLE
Settings Themes

### PR DESCRIPTION
**Pull request by Serginho6**
- StatusBar isn't responsive with themes yet, let it black for both.
- SettingsFragment now has icons responsive to selected theme.
- SiginIn materialCardviews margin fixed

### CODE REVIEW

File: app/src/main/java/com/app/fitspace/presentation/fragment/SettingsFragment.kt [hunk 1]
```diff
@@ -42,6 +43,39 @@ class SettingsFragment : Fragment() {
         return rootView
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val profileIcon: ImageView = view.findViewById(R.id.image_view_item_profile)
```
Comment (jbalves) [inline app/src/main/java/com/app/fitspace/presentation/fragment/SettingsFragment.kt]: o que você acha de utilizarmos Viewbinding para acessar esses componentes de UI ?

**Review by jbalves — CHANGES_REQUESTED** [review]
favor verificar a viabilidade de utilizar Viewbindig

File: app/src/main/java/com/app/fitspace/presentation/fragment/SettingsFragment.kt (hunk 1 above)
Comment (Serginho6) [reply app/src/main/java/com/app/fitspace/presentation/fragment/SettingsFragment.kt]: Fiz a refatoração da SettingsFragment e do HomeFragment para utilizar o Viewbinding.

**Review by jbalves — APPROVED** [review]
Great Job!

**Pull request merged**